{"name": "登录异常，测试数据是：{'accounts': 'lee11', 'pwd': '111111', 'exp': '密码错误'}", "status": "passed", "parameters": [{"name": "args", "value": "{'accounts': 'lee11', 'pwd': '111111', 'exp': '密码错误'}"}], "start": 1629534565933, "stop": 1629534566058, "uuid": "24896e47-4fc3-4d5f-9ec3-a1015781224e", "historyId": "811b4a31d3a0cf9976694185c00512b4", "testCaseId": "ec1071642d55a2fafea25ec18572e772", "fullName": "script.test_mtx_login.TestLogin#test_login_fail", "labels": [{"name": "parentSuite", "value": "script"}, {"name": "suite", "value": "test_mtx_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "WIN-0A2AUJ2TO0H"}, {"name": "thread", "value": "16056-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "script.test_mtx_login"}]}